dashboard/app/views/layouts/_header.html.haml

```diff
@@ -12,6 +12,15 @@
 - show_bug_links = (script_level && script.show_report_bug_link?) || (level && level.try(:is_project_level))
 
 .header-wrapper.hide_on_mobile{class: ('feedback-bug-links' if show_bug_links)}
+  - if current_user && script_level
+    - post_milestone = script ? Gatekeeper.allows('postMilestone', where: {script_name: script.name}, default: true) : true
+    - if ! post_milestone
```

Comment:
style is to not have space after !
you can also use unless (I'm not that into unless)